sdk/tables/azure-data-tables/azure/data/tables/_constants.py

```diff
@@ -5,7 +5,7 @@
 # --------------------------------------------------------------------------
 import platform
 import sys
-from ._generated.version import VERSION
+from ._generated._version import VERSION
 
 # UserAgent string sample: 'Azure-Storage/0.37.0-0.38.0 (Python CPython 3.4.2; Windows 8)'
 # First version(0.37.0) is the common package, and the second version(0.38.0) is the service package
```

Comment:
Are we actually using the user-agent string below? This doesn't fit our format:
https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy

Reply:
Also we've talked about this partially offline, @seankane-msft were you able to find out a way to not import this and set the x-ms-version on our side, and just rely on the generated code? I think this can also be addressed in a separate PR as long as there's an issue open for it

Reply:
I have not yet, there is an issue out related with the versioning (#12411)